scripts/packages/VSCodeServer/src/display.jl

```diff
@@ -1,105 +1,50 @@
 struct InlineDisplay <: AbstractDisplay end
 
-function sendDisplayMsg(kind, data)
-    JSONRPC.send_notification(conn_endpoint[], "display", Dict{String,String}("kind" => kind, "data" => data))
-end
+const PLOT_PANE_ENABLED = Ref(true)
 
-
-function display(d::InlineDisplay, ::MIME{Symbol("image/png")}, x)
-    payload = stringmime(MIME("image/png"), x)
-    sendDisplayMsg("image/png", payload)
+function toggle_plot_pane(_, enable::Bool)
+    PLOT_PANE_ENABLED[] = enable
 end
 
-displayable(d::InlineDisplay, ::MIME{Symbol("image/png")}) = true
-
-function display(d::InlineDisplay, ::MIME{Symbol("image/gif")}, x)
-    payload = stringmime(MIME("image/gif"), x)
-    sendDisplayMsg("image/gif", payload)
-end
-
-displayable(d::InlineDisplay, ::MIME{Symbol("image/gif")}) = true
-
-function display(d::InlineDisplay, ::MIME{Symbol("image/svg+xml")}, x)
-    payload = stringmime(MIME("image/svg+xml"), x)
-    sendDisplayMsg("image/svg+xml", payload)
+function fix_displays()
+    for d in reverse(Base.Multimedia.displays)
+        if d isa InlineDisplay
+            popdisplay(InlineDisplay())
+        end
+    end
+    pushdisplay(InlineDisplay())
 end
 
-displayable(d::InlineDisplay, ::MIME{Symbol("image/svg+xml")}) = true
-
-function display(d::InlineDisplay, ::MIME{Symbol("text/html")}, x)
-    payload = stringmime(MIME("text/html"), x)
-    sendDisplayMsg("text/html", payload)
+function sendDisplayMsg(kind, data)
+    JSONRPC.send_notification(conn_endpoint[], "display", Dict{String,String}("kind" => kind, "data" => data))
 end
 
-displayable(d::InlineDisplay, ::MIME{Symbol("text/html")}) = true
-
-function display(d::InlineDisplay, ::MIME{Symbol("juliavscode/html")}, x)
-    payload = stringmime(MIME("juliavscode/html"), x)
-    sendDisplayMsg("juliavscode/html", payload)
+function display(_::InlineDisplay, m::MIME, x)
```

Comment:
```suggestion
function display(d::InlineDisplay, m::MIME, x)
```

cause we may throw an err in `throw(MethodError(display, (d, m, x)))`